fem/pfespace.cpp

```diff
@@ -2190,6 +2403,7 @@ ParFiniteElementSpace::ParallelDerefinementMatrix(int old_ndofs,
         it = col_map.begin(); it != col_map.end(); ++it)
    {
       cmap[it->second-1] = it->first;
+      // FIXME: it seems cmap is not monotonic here, hypre does not like that
```

Comment:
Is this still an issue? Does it mean that parallel derefinement does not work?

Reply:
Parallel derefinement works, despite the comment. It's just something that could potentially break things, maybe we should make the array monotonic (but I'm not sure really).

Reply:
As far as I know, hypre expects the column-map for the off-diagonal matrix to be sorted.

Reply:
OK, I will fix it.